src/internal/operations/restore.go

```diff
@@ -274,34 +286,34 @@ func formatDetailsForRestoration(
 	ctx context.Context,
 	sel selectors.Selector,
 	deets *details.Details,
+	errs fault.Adder,
 ) ([]path.Path, error) {
 	fds, err := sel.Reduce(ctx, deets)
 	if err != nil {
 		return nil, err
 	}
 
 	var (
-		errs     *multierror.Error
 		fdsPaths = fds.Paths()
 		paths    = make([]path.Path, len(fdsPaths))
 	)
 
 	for i := range fdsPaths {
 		p, err := path.FromDataLayerPath(fdsPaths[i], true)
 		if err != nil {
-			errs = multierror.Append(
-				errs,
-				errors.Wrap(err, "parsing details entry path"),
-			)
+			errs.Add(clues.
+				Wrap(err, "parsing details path after reduction").
+				WithMap(clues.Values(ctx)).
+				With("path", fdsPaths[i]))
 
 			continue
 		}
 
 		paths[i] = p
 	}
 
-	if errs != nil {
-		return nil, errs
+	if err != nil {
```

Comment:
Is this not a typo? `errs` is updated with clues and then it is not returned or checked if it is nil at L315

Reply:
it is, good catch.